corehq/apps/linked_domain/README.md

```diff
@@ -36,15 +36,16 @@ supported applications. Remote domain linkages cannot be created via the UI; see
 ## Data models
 
 Linked domains share configuration data. Supported data types are defined in
-[LINKED_MODELS](https://github.com/dimagi/commcare-hq/blob/966b62cc113b56af771906def76833446b4ba025/corehq/apps/linked_domain/const.py#L13):
+[ALL_LINKED_MODELS](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/linked_domain/const.py#L43):
```

Comment:
It's a good catch to update this, but the line number is likely to get out of date. Maybe instead of a link, this doc could just say `corehq.apps.linked_domain.const.ALL_LINKED_MODELS` or even just `ALL_LINKED_MODELS`? It should be easy enough for readers to grep for the definition.